exporters/metrics/src/main/java/com.google.cloud.opentelemetry.metric/MetricTranslator.java

```diff
@@ -35,6 +37,8 @@
   private static final Logger logger = LoggerFactory.getLogger(MetricTranslator.class);
 
   static final String DESCRIPTOR_TYPE_URL = "custom.googleapis.com/OpenTelemetry/";
+  static final List<String> KNOWN_DOMAINS = ImmutableList
```

Comment:
Should this be a `Set<>` for faster lookups?

Reply:
Yes! 😁